src/Net/Http.fs

```diff
@@ -1145,6 +1145,7 @@ module private HttpHelpers =
         }
 
     let writeBody (req:HttpWebRequest) (data: Stream) =
+#if FX_NO_LOCAL_FILESYSTEM
```

Comment:
This seems unrelated to not having local filesystem, should a separate FX_xxx be added?